com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/PyFunctionHashRemover/PyFunctionHashRemover.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" ?>
+<operatorModel
```

Comment:
I don't understand why we need a new operator to remove an attribute. Just have a Filter with a different schema on the output from the input.